# S106 - Invalid Literal Argument

<!-- This file is automatically generated from diagnostics.toml -->
<!-- Do not edit manually. To update this rule, edit the TOML file instead. -->

**Code:** `S106`  
**Category:** semantic  
**Severity:** error  

## Description

Template tag argument must be a specific literal value

## Details

This error occurs when a template tag expects a specific literal keyword but receives
something else. Some Django tags require exact keyword matches at specific positions.

Example:
```django
{% for item on items %}  {# Should be 'in', not 'on' #}
    {{ item }}
{% endfor %}

{% if user is_not active %}  {# Should be 'not', not 'is_not' #}
    ...
{% endif %}
```

Fix:
```django
{% for item in items %}
    {{ item }}
{% endfor %}

{% if user is not active %}
    ...
{% endif %}
```
