# S107 - Invalid Argument Choice

<!-- This file is automatically generated from diagnostics.toml -->
<!-- Do not edit manually. To update this rule, edit the TOML file instead. -->

**Code:** `S107`  
**Category:** semantic  
**Severity:** error  

## Description

Template tag argument must be one of a specific set of choices

## Details

This error occurs when a template tag argument must be one of a predefined set of
values, but receives something else. Unlike S106 which expects a single literal,
this applies when multiple valid options exist.

Example:
```django
{% cycle 'odd' 'even' as rowcolors silent=true %}  {# invalid; should be 'silent' without '=' #}

{% autoescape yes %}  {# valid #}
{% autoescape maybe %}  {# invalid - must be 'on' or 'off' #}
```

Fix:
```django
{% cycle 'odd' 'even' as rowcolors silent %}
{% autoescape on %}
```
