# T100 - Parser Error

<!-- This file is automatically generated from diagnostics.toml -->
<!-- Do not edit manually. To update this rule, edit the TOML file instead. -->

**Code:** `T100`  
**Category:** template  
**Severity:** error  

## Description

Template syntax error encountered during parsing

## Details

This error occurs when the template parser encounters invalid Django template syntax.
Common causes include:
- Malformed template tags (e.g., missing closing braces)
- Invalid variable syntax
- Unclosed strings or comments
- Unexpected characters in template expressions

Example:
```django
{# This will cause a parser error
{{ variable | filter: invalid }}
```
